src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.c

```diff
@@ -1323,27 +1324,27 @@ void EnSyatekiMan_Town_RunGame(EnSyatekiMan* this, PlayState* play) {
             this->perGameVar1.octorokState = SG_OCTO_STATE_HIDING;
         }
 
-        if (this->perGameVar2.octorokHitType != SG_OCTO_HIT_TYPE_NONE) {
-            if (this->perGameVar2.octorokHitType == SG_OCTO_HIT_TYPE_BLUE) {
+        if (this->perGameVar2.octorokHitType != SG_OCTO_TYPE_NONE) {
+            if (this->perGameVar2.octorokHitType == SG_OCTO_TYPE_BLUE) {
                 gSaveContext.timerTimeLimits[TIMER_ID_MINIGAME_1] -= SECONDS_TO_TIMER_PRECISE(2, 50);
                 sModFromLosingTime = (sModFromLosingTime + 25) % 50;
             }
 
-            this->perGameVar2.octorokHitType = SG_OCTO_HIT_TYPE_NONE;
+            this->perGameVar2.octorokHitType = SG_OCTO_TYPE_NONE;
```

Comment:
```suggestion
            this->perGameVar2.octorokType = SG_OCTO_TYPE_NONE;
```
Should this reflect dropping `HIT` in the enum as well?

Reply:
No, I don't think we want to do this, though I think this name can definitely be improved. What this variable represents is the type of the last Octorok that was hit by the player. This block of code runs every single frame the minigame is running:
```c
        if (this->perGameVar2.octorokHitType != SG_OCTO_TYPE_NONE) {
            if (this->perGameVar2.octorokHitType == SG_OCTO_TYPE_BLUE) {
                gSaveContext.timerTimeLimits[TIMER_ID_MINIGAME_1] -= SECONDS_TO_TIMER_PRECISE(2, 50);
                sModFromLosingTime = (sModFromLosingTime + 25) % 50;
            }

            this->perGameVar2.octorokHitType = SG_OCTO_TYPE_NONE;
        }
```

And every time an Octorok *gets* hit, it does this:
```c
    if (EnSyatekiOkuta_CheckCollision(this, play)) {
        syatekiMan = (EnSyatekiMan*)this->actor.parent;
        if (this->type == SG_OCTO_TYPE_RED) {
            Actor_PlaySfx(&this->actor, NA_SE_SY_TRE_BOX_APPEAR);
            play->interfaceCtx.minigamePoints++;
            syatekiMan->score++;
            syatekiMan->perGameVar2.octorokHitType = SG_OCTO_TYPE_RED;
        } else {
            Actor_PlaySfx(&this->actor, NA_SE_SY_ERROR);
            syatekiMan->perGameVar2.octorokHitType = SG_OCTO_TYPE_BLUE;
        }

        EnSyatekiOkuta_SetupDie(this);
    } 
```

In other words, every time the player hits an Octorok, it updates a variable on the Shooting Gallery Man. He then checks to see if it was a blue Octorok, and if so, reduces the timer a bit. No matter what, he clears the variable so that the next time an Octorok gets hit, he'll be aware of it.

Maybe something like `lastHitOctorokType` would work here? But we definitely don't want to just do `octorokType`, since that's too vague as to what the purpose of this variable is. Hopefully, that helps explain it!

Reply:
As I was reading that, `lastHitOctorokType` was the name that came to mind before I got to the end, so that's a good sign. Works for me